spring-cloud-netflix-core/pom.xml

```diff
@@ -69,6 +69,10 @@
 			<artifactId>archaius-core</artifactId>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
```

Comment:
No, you can use Spring Beans and check is the `Pageable` class is available and then include/exclude encoders based on that.